courier/stub/request.config.mailer.jsonnet

```diff
@@ -0,0 +1,11 @@
+function(ctx) {
```

Comment:
This looks like a good default value for the body payload. Maybe it makes sense to use it as the default?

Reply:
Not sure how we'd make it the default as the body of the template engine is a file URI not a absolute value?